app/views/key_pairs/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<% title _("Key pairs for: %s") % @compute_resource.name %>
+
+<table class="<%= table_css_classes 'table-fixed' %>">
+  <% if @key_pairs.any? %>
+  <thead>
+  <tr>
+    <th width="6%"><%= _('Active key') %></th>
+    <th width="40%"><%= _('Name') %></th>
```

Comment:
![image](https://cloud.githubusercontent.com/assets/433583/20883847/da16563e-baf0-11e6-8331-01144eb25af2.png)

It may be only me, but this itches me :)
Previous solution works better, maybe only change the title (or remove the title completely)?